src/parsehdr.c

```diff
@@ -33,6 +33,7 @@
 
 #ifdef ENABLE_LEGACY_WEAKDEPS
 #define RPMSENSE_STRONG (1 << 27)
+#define RPMSENSE_MISSINGOK (1 << 19)
```

Comment:
Not a new issue, but why aren't we doing `#include <rpm/rpmds.h>`?  So we can build with older rpm?

Reply:
I imagine so. `createrepo_c` supports RHEL 6 and newer.